terraform-environments/aws/terragrunt-dev/us-east-1/terragrunt-dev/200-eks/terragrunt.hcl

```diff
@@ -101,7 +101,7 @@ inputs = {
       desired_size   = 2
       max_size       = 3
       min_size       = 0
-      instance_types = ["t3.small"]
+      instance_types = ["t3.medium"]
```

Comment:
Increasing the size of the nodes because istio needs 2GB of memory and the `t3.small` only has 2GB.  Increasing to a node with 4GB of memory

https://github.com/ManagedKube/kubernetes-ops/pull/300